docs.kosli.com/content/integrations/sonar.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Sonar
+bookCollapseSection: false
+weight: 340
+---
+# Sonar in Kosli
+
+The results of SonarCloud and SonarQube scans can be tracked in [Kosli trails](/getting_started/trails/).
+
+## Setting up in Kosli
+
+To set up the integration, navigate to the [Sonar integration page](https://app.kosli.com/cyber-dojo/integrations/sonar/) for your org in the [Kosli app](https://app.kosli.com/).
```

Comment:
the first link is for cyber-dojo. You could point them to the integrations menu instead